cylc/flow/network/schema.py

```diff
@@ -1460,7 +1460,7 @@ class Flow(String):
 class Broadcast(Mutation):
     class Meta:
         description = sstrip('''
-            Override `[runtime]` configurations in a running workflow.
+            Override `[runtime]` configurations in a workflow.
```

Comment:
Now the `Valid for:` bit is at the bottom the `running` here is useful again.

```suggestion
            Override `[runtime]` configurations in a running workflow.
```